widgy/contrib/page_builder/models.py

```diff
@@ -485,7 +485,7 @@ class Figure(StrDisplayNameMixin, Content):
     tooltip = _("A figure is a self-contained piece of content. It can be used"
                 " to add a caption to an image or a video for example.")
 
-    position = models.CharField(default=lambda: ugettext('center'), verbose_name=_('position'), max_length=50, choices=[
+    position = models.CharField(default=_('center'), verbose_name=_('position'), max_length=50, choices=[
```

Comment:
Pure question, not a remark. I don't understand this change. Why was it a lambda? Why isn't it a lambda anymore?